<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="DRAW">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="DRAWBAR">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="REGEN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DRAW</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="batInd">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;PibotoLt&quot;;
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>1.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>REGEN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="M1TEMP">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="INV1TEMP">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="SPD">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="SPDBAR">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>kW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>kW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>101</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>km/hr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>m1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>inv1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>220</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>310</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="REGENBAR">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="INVERTERFAULT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FAULT: INVERTER TEMP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="MOTORTEMPFAULT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FAULT: MOTOR TEMP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="CANSHORTFAULT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FAULT: CAN SHORT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="HIGHVOLTAGEFAULT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FAULT: HIGH VOLTAGE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OPENCANFAULT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FAULT: CAN OPEN CIRCUIT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>DRAW</zorder>
   <zorder>DRAWBAR</zorder>
   <zorder>REGEN</zorder>
   <zorder>label_2</zorder>
   <zorder>batInd</zorder>
   <zorder>label_4</zorder>
   <zorder>M1TEMP</zorder>
   <zorder>INV1TEMP</zorder>
   <zorder>SPD</zorder>
   <zorder>SPDBAR</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>REGENBAR</zorder>
   <zorder>INVERTERFAULT</zorder>
   <zorder>MOTORTEMPFAULT</zorder>
   <zorder>CANSHORTFAULT</zorder>
   <zorder>HIGHVOLTAGEFAULT</zorder>
   <zorder>OPENCANFAULT</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
